Makefile

```diff
@@ -49,13 +49,11 @@ endif
 # Set the Operator SDK version to use. By default, what is installed on the system is used.
 # This is useful for CI or a project to utilize a specific version of the operator-sdk toolkit.
 OPERATOR_SDK_VERSION ?= v1.39.2
-# Image URL to use all building/pushing image targets
-IMG ?= controller:latest
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
 ENVTEST_K8S_VERSION = 1.31.0
 
 # Image URL to use all building/pushing image targets
-IMG ?= quay.io/opendatahub/llama-stack-k8s-operator:latest
+IMG ?= quay.io/llamastack/llama-stack-k8s-operator:latest
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
 ENVTEST_K8S_VERSION = 1.24.2
```

Comment:
Please remove this var, too. My bad, this slipped in during a conflict resolution when rebasing.

Reply:
I think this should be okay to point to default image if we want to just do a simple deploy using `make deploy` without any custom image

Reply:
Sorry, I meant `ENVTEST_K8S_VERSION` as it appears twice in the Makefile with different values.

Reply:
Updated @rhuss